docs/snippets/GPU_Kernels_Tuning.cpp

```diff
@@ -4,9 +4,11 @@
 int main() {
 using namespace InferenceEngine;
 //! [part0]
-Core ie;          
+Core ie;
+IE_SUPPRESS_DEPRECATED_START
   ie.SetConfig({{ CONFIG_KEY(TUNING_MODE), CONFIG_VALUE(TUNING_CREATE) }}, "GPU");
   ie.SetConfig({{ CONFIG_KEY(TUNING_FILE), "/path/to/tuning/file.json" }}, "GPU");
+IE_SUPPRESS_DEPRECATED_END
```

Comment:
I think snippets should demonstrate new API only. If we deprecated this option, maybe better to remove its demonstration to users.

Reply:
This snipped was removed